style.css

```diff
@@ -0,0 +1,282 @@
+html {
+  font-size: 62.5%;
+  text-rendering: optimizeLegibility;
```

Comment:
What's this? Looks cool

Reply:
Honestly, I googled the best way to do full cross browser font smoothing. I suspect this covers a weird Windows case or something similar